ably/realtime/connection.py

```diff
@@ -144,6 +145,7 @@ def __init__(self, realtime, initial_state):
         self.__websocket = None
         self.setup_ws_task = None
         self.__ping_future = None
+        self.timeout_in_secs = self.options.realtime_request_timeout / 1000
```

Comment:
This field is internal so should be `self.__timeout_in_secs`. In fact, I think it would be better to just divide it by 1000  when you're passing it into `wait_for`, ie `await asyncio.wait_for(self.__connected_future, self.options.realtime_request_timeout / 1000)` (same goes for `RealtimeChannel`)

Reply:
It might be a bit tedious and much repetition having to put it all the `wait_for`. Might be better to make it internal instead

Reply:
I don't think there's much difference either way tbh, so I'm happy for you to just make it internal if you prefer :)